### PR TITLE
Issue #1486: Clarify hierarchy structure of inheritance terms

### PR DESCRIPTION
**Pull request by danielpgross**
Done

### CODE REVIEW

**Comment by danielpgross** [comment]
p0000001___phenotype_database

**Comment by buske** [comment]
So much clearer!

File: components/ontology-access/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml [hunk 1]
```diff
@@ -76,7 +76,15 @@ $xwiki.jsx.use('PhenoTips.HPOBranchDisplayer')##
...
     #set ($foreach.item = $macro.childrenMap.get($orderedItemId))
-    #if ("$!{foreach.item}" != '' &amp;&amp; "$!foreach.item.id" != '')&lt;li class="term-entry"&gt;#_displayItem("${foreach.item.id}", "${foreach.item.name}", $foreach.item)#_displayChildren($foreach.item.id)&lt;/li&gt;#end
+    #if ("$!{foreach.item}" != '' &amp;&amp; "$!foreach.item.id" != '')
+      #set ($childrenHtml = "#_displayChildren($foreach.item.id)")
+      #if ("$childrenHtml" != "")
+        #set ($childrenExpandSymbol = '&lt;span class="expand-tool locked">▼&lt;/span>')
+      #else
+        #set ($childrenExpandSymbol = '&lt;span class="expand-tool locked disabled">▼&lt;/span>')
+      #end
+      &lt;li class="term-entry">$!childrenExpandSymbol#_displayItem("${foreach.item.id}", "${foreach.item.name}", $foreach.item)$!childrenHtml&lt;/li>
```
Comment (sdumitriu) [inline components/ontology-access/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: `>` should also be escaped as `&gt;`

**Comment by sdumitriu** [comment]
I find it a bit weird that for radio buttons `Unknown` is pulled to the left:

File: components/ontology-access/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml (hunk 1 above)
Comment (danielpgross) [inline components/ontology-access/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: Ok, I will change it. FYI, this is the output from the xar export.

Comment (sdumitriu) [inline components/ontology-access/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: Yes, but we're supposed to run `mvn xar:format ; mvn xar:format` in the module with the changes before committing. That's an XWiki annoyance, which I think might be fixed in a more recent version.

Comment (danielpgross) [inline components/ontology-access/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: Ah I see, thanks.

**Comment by danielpgross** [comment]
There's room to say that ```Unknown``` is different from the rest and should therefore appear separate, but for now I'll align it with the rest of the items.

**Comment by danielpgross** [comment]
I see now that there already is a small ```margin-bottom``` applied to the ```Unknown``` item to separate it, and I agree that this looks much better:
p0000001___phenotype_database

**Pull request merged**